Katas/src/at/codersbay/katas/ArrayMean.java

```diff
@@ -0,0 +1,32 @@
+package at.codersbay.katas;
+
+public  class ArrayMean {
+
+
+	
+	public static void main(String[] args) {
+		
+	
+		calcArravg();
```

Comment:
according to JavaCodeConvention please rename your method to calcArrAvg()

Reply:
> 
> 
> according to JavaCodeConvention please rename your method to calcArrAvg()

done.